.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: rust
+rust: nightly
+sudo: true
+before_script:
+  - sudo apt-get install --assume-yes libfftw3-dev
```

Comment:
I think this can be installed via, which allows `sudo` to be switched to `sudo: false`.

```rust
addons:
  apt:
    packages:
      - libfftw3-dev
```